docs/agents/cryomech_cpa1114.rst

```diff
@@ -0,0 +1,68 @@
+.. highlight:: rst
+
+.. _cryomech_cpa1114:
+
+======================
+Cryomech CPA1114 Agent
+======================
+
+The Cryomech CPA1114 compressor is a commonly used compressor model for the
+pulse tubes within SO. The CPA1114 Agent interfaces with the compressor over
+ethernet to monitor the health of the unit, including stats such as Helium
+temperature and pressure, oil temperature, and more. Control is not yet
+implemented.
+
+.. argparse::
+    :filename: ../agents/cryomech_cpa1114/cryomech_cpa1114_agent.py
+    :func: make_parser
+    :prog: python3 crypmech_cpa1114_agent.py
+
+Description
```

Comment:
Was there supposed to be text underneath this header?

Reply:
Yeah, I think I copied the header structure from the [timestream aggregator page](https://socs.readthedocs.io/en/latest/agents/timestream_aggregator.html). If there isn't any description to add beyond the initial one we could probably remove this header (no other pages have it.)